beeline.go

```diff
@@ -96,16 +99,49 @@ type Config struct {
 	PprofTagging bool
 }
 
+func IsLegacyKey(config Config) bool {
```

Comment:
Update to Classic?

![hashtag classic](https://user-images.githubusercontent.com/517302/155799955-3074f10c-cc03-456b-a8bb-d679431d515f.gif)

Reply:
Definitely! https://github.com/honeycombio/beeline-go/pull/309/commits/ccc20ea5ffebe89784928d256d722a09c8b2b25c